packages/quantic/cypress/integration/tab/tab.cypress.ts

```diff
@@ -6,6 +6,7 @@ import {
 } from '../../page-objects/search';
 import {TabExpectations as Expect} from './tab-expectations';
 import {TabActions as Actions} from './tab-actions';
+import {TabSelector, TabSelectors} from './tab-selectors';
```

Comment:
You are not using these in this file. You need to remove the unused imports.

Reply:
unused imports removed